drake_ament_cmake_installed/src/drake_example_pendulum_description/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Can we stick with CMake 3.10 for consistency. And can you add the license block.